src/docs/asciidoc/api-guide/overview/index.adoc

```diff
@@ -9,3 +9,5 @@ include::http-status-codes.adoc[]
 include::hypermedia.adoc[]
 
 include::errors.adoc[]
+
+include::terms-of-service.adoc[]
```

Comment:
I think it should live at straight the top to be more visible (before HTTP verbs).